client/coral-admin/src/AppRouter.js

```diff
@@ -35,10 +35,14 @@ const routes = (
       {/* Moderation Routes */}
 
       <Route path='moderate' component={ModerationLayout}>
+
```

Comment:
New IndexRoute should exist here to route to `Moderation` e.g. `<IndexRoute component={Moderation} />`.